antora-ui-camel/src/css/frontpage.css

```diff
@@ -235,6 +235,7 @@ section.frontpage h1 {
 @media screen and (max-width: 626px) {
   .split {
     flex: 100%;
+    margin: 3%;
```

Comment:
Perhaps a fixed margin would work better?

Reply:
I would try it to fix it with rem and used a fixed margin.

Reply:
I used a fixed margin by adjusting the **margin-left**.